<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigsObjDadosGeraisWidget</class>
 <widget class="QWidget" name="ConfigsObjDadosGeraisWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>350</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Dados Gerais</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Tipo:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QComboBox" name="comboTipo">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>Personagem - Oito Direções</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/icones_movimentos/data/movement_icons/eightmovement.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Personagem - Carro</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/icones_movimentos/data/movement_icons/carmovement.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Personagem - Plataforma</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/icones_movimentos/data/movement_icons/platformer.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Personagem - Luta</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/icones_movimentos/data/movement_icons/fighting_movement.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Personagem - Bola de Pong</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/icones_movimentos/data/movement_icons/bouncing_movement.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Personagem - Segue mouse</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/icones_movimentos/data/movement_icons/mouse_movement.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Objeto - Estático</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/icones_movimentos/data/movement_icons/static_movement.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Aliado</string>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/icones_movimentos/data/movement_icons/npc_icon.png</normaloff>:/icones_movimentos/data/movement_icons/npc_icon.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Inimigo</string>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/icones_movimentos/data/movement_icons/inimigo_icon.png</normaloff>:/icones_movimentos/data/movement_icons/inimigo_icon.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Item</string>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/icones_movimentos/data/movement_icons/item_icon.png</normaloff>:/icones_movimentos/data/movement_icons/item_icon.png</iconset>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="3" rowspan="2">
       <widget class="QGraphicsView" name="graphicsViewSprite"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Nome:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="editNome"/>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QCheckBox" name="checkBoxObjUnico">
        <property name="text">
         <string>Objeto único no cenário</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QLabel" name="labelFormaGrafica">
        <property name="text">
         <string>Forma gráfica:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QComboBox" name="comboFormaGrafica">
        <item>
         <property name="text">
          <string>Sprite único</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sprite único que gira</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Animação</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Animação que gira</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="botaoEditarFormaGrafica">
        <property name="text">
         <string>Editar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="checkBoxDrag">
     <property name="text">
      <string>Pode ser arrastado pelo mouse?</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
